src/forms/_input_group.scss

```diff
@@ -0,0 +1,63 @@
+// Input group has an input and a button_one
+// If the button has text, it is alongside the input field, if it has an icon, it is
+// positioned absolutely on top
+
```

Comment:
We'll want to continue the JSDoc style docs at the top of the page. This description is great for @overview directive. One fix would be to just use `button` instead of `button_one`